### PR TITLE
debug all playbooks and added them to "make jenkins-ansible-playbook"

### PR DESCRIPTION
**Pull request by feifanya**
1. *.yaml;
2. main.yaml,
3. README
4.Dockerfile

### CODE REVIEW

File: module_utils/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (ronanjs) [inline module_utils/.vscode/launch.json]: remove this file (at .vscode to .gitignore)

File: module_utils/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (ronanjs) [inline module_utils/.vscode/settings.json]: remove this file (at .vscode to .gitignore)

File: module_utils/xpath_test.py [hunk 3]
```diff
@@ -144,7 +163,7 @@ def test1d(self):
         nodes = linker._resolve('/port[name="port 1"]')
         assert nodes.count() == 1 and nodes.get(0) == self.port1
 
-    def test1d(self):
+    def test1e(self):
```
Comment (ronanjs) [inline module_utils/xpath_test.py]: why?

File: playbooks/bgp-traffic-modify.yaml [hunk 4]
```diff
@@ -220,12 +219,10 @@
...
       - StreamBlock: 
           TrafficPattern: PAIR
-          Name: BGPTaffic1
+          name: BGPTaffic1
           EnableStreamOnlyGeneration: true
-          SrcBinding-targets: ref:/EmulatedDevice[Name=BGPDevice1]/BgpRouterConfig[Name=BGPRouter1]/BgpIpv4RouteConfig/Ipv4NetworkBlock
-          DstBinding-targets: ref:/EmulatedDevice[Name=BGPDevice2]/BgpRouterConfig[Name=BGPRouter2]/BgpIpv4RouteConfig/Ipv4NetworkBlock
-          AffiliationStreamBlockLoadProfile: 
-            Load: 10
+          SrcBinding-targets: ref:/EmulatedDevice[Name=BGPDevice1]/Ipv4If
```
Comment (ronanjs) [inline playbooks/bgp-traffic-modify.yaml]: Why?

File: inventory.ini [hunk 5]
```diff
@@ -1,7 +1,7 @@
 [labservers]
 cal-ls ansible_host=10.109.127.125 chassis="10.109.123.236 10.109.123.181"
 bdc-ls ansible_host=10.61.67.200 chassis="10.61.67.128 10.61.67.131"
-rtp-ls ansible_host=10.61.67.200 chassis="10.28.236.32 10.28.236.36"
+rtp-ls ansible_host=10.28.238.4 chassis="10.28.236.251 10.28.236.28"
```
Comment (ronanjs) [inline inventory.ini]: You should use mintaka to dynamically update the IP addresses

File: module_utils/.vscode/launch.json (hunk 1 above)
Comment (feifanya) [reply module_utils/.vscode/launch.json]: okay, will remove it through .gitignore

File: module_utils/.vscode/settings.json (hunk 2 above)
Comment (feifanya) [reply module_utils/.vscode/settings.json]: okay, will remove it through .gitignore

File: module_utils/xpath_test.py (hunk 3 above)
Comment (feifanya) [reply module_utils/xpath_test.py]: there are 2 functions with same name test1d(self), so rename one test1d() to test1e()

File: playbooks/bgp-traffic-modify.yaml (hunk 4 above)
Comment (feifanya) [reply playbooks/bgp-traffic-modify.yaml]: will change binding back

File: inventory.ini (hunk 5 above)
Comment (feifanya) [reply inventory.ini]: modify mintaka.py to implement one ansible var plugin to support this feature

**Pull request merged**